<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FqFitAddWorkspaceDialog</class>
 <widget class="QDialog" name="FqFitAddWorkspaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JumpFit Data Selection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbSelectData">
     <property name="title">
      <string>Select data</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="loInput" rowstretch="0,0,0">
        <item row="0" column="1">
         <widget class="MantidQt::MantidWidgets::DataSelector" name="dsWorkspace" native="true">
          <property name="showLoad" stdset="0">
           <bool>false</bool>
          </property>
          <property name="ShowGroups" stdset="0">
           <bool>false</bool>
          </property>
          <property name="autoLoad" stdset="0">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="cbParameterName"/>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cbParameterType">
          <item>
           <property name="text">
            <string>Width</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>EISF</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lbParameterType">
          <property name="text">
           <string>Parameter Type</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lbParameterName">
          <property name="text">
           <string>Parameter Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lbWorkspace">
          <property name="text">
           <string>Workspace</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>147</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbAdd">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbClose">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::DataSelector</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Common/DataSelector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pbAdd</sender>
   <signal>clicked()</signal>
   <receiver>FqFitAddWorkspaceDialog</receiver>
   <slot>addData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbClose</sender>
   <signal>clicked()</signal>
   <receiver>FqFitAddWorkspaceDialog</receiver>
   <slot>closeDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addData()</slot>
  <slot>closeDialog()</slot>
 </slots>
</ui>
